### PR TITLE
Cu 86bx53yj6 weekend meeting ilona

### PR DESCRIPTION
**Pull request by IlkaA1**
Add other weekType and footer on weekendMeeting page.

## Summary by CodeRabbit

- **New Features**
    - Introduced a PDF viewer feature for displaying weekend meeting schedules.
    - Added a new route `/pdf-document` for accessing the PDF viewer in the application.
    - Implemented dynamic generation of weekend meeting data for PDF document formatting.
    - Added specific components for rendering the weekend meeting PDF, including a custom SVG icon, header, footer, and meeting items.

### CODE REVIEW

**Comment by rhahao** [comment]
Task linked: [CU-86bx53yj6 Weekend Meeting](https://app.clickup.com/t/86bx53yj6)

**Comment by rhahao** [comment]
Thanks @IlkaA1, also for looking at these merge conflicts. 😊

**Comment by IlkaA1** [comment]
> Thanks @IlkaA1, also for looking at these merge conflicts. 😊

Thank you☺️! Can you look which conflicts is in security/snyk?

**Comment by rhahao** [comment]
Try to fix those issues first. I hope the merge conflict in Snyk will be solved with that as well.

File: src/v3/PDF_Peview.tsx
Comment (rhahao) [inline src/v3/PDF_Peview.tsx:1]: @IlkaA1: could you please move this file back to `v3/components/preview` folder?

**Review by rhahao — CHANGES_REQUESTED** [review]

**Pull request merged**